<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessingSettingsDialog</class>
 <widget class="QDialog" name="ImageProcessingSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processing Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_40">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="overlaytab">
       <attribute name="title">
        <string>Overlays</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget2_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>401</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_41">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_48">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_42">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout"/>
             </item>
             <item>
              <widget class="QLabel" name="label_68">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>27</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Type:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_43">
               <item>
                <widget class="QCheckBox" name="glasses">
                 <property name="text">
                  <string>Glasses</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="nose">
                 <property name="text">
                  <string>Nose</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="redlips">
                 <property name="text">
                  <string>Red Lips</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_29">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showFacedetection">
           <property name="text">
            <string>Show facedetection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showOverlays">
           <property name="text">
            <string>Show Overlays</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_31">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resetOverlays">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_59">
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you are in the &amp;quot;Single Image Mode&amp;quot; then the apply function only updates the stored settings. To really apply hit &amp;quot;OK&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="okCancelBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>okCancelBox</tabstop>
  <tabstop>resetOverlays</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageProcessingSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageProcessingSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
